htdocs/societe/class/societe.class.php

```diff
@@ -1388,12 +1388,7 @@ public function update($id, $user = '', $call_trigger = 1, $allowmodcodeclient =
 		$this->localtax1_value = trim($this->localtax1_value);
 		$this->localtax2_value = trim($this->localtax2_value);
 
-		if ($this->capital != '') {
-			$this->capital = price2num(trim($this->capital));
-		}
-		if (!is_numeric($this->capital)) {
-			$this->capital = ''; // '' = undef
-		}
+		$this->capital = (!empty($this->capital)) ? (float) price2num(trim($this->capital)) : 0;
```

Comment:
We must be able to set nothing (so '' in input = null in databse) OR 0 OR a value.
It seems this break this feature...

Reply:
Yes I agree. I will refactor.